web/source/kmwcallback.ts

```diff
@@ -42,6 +42,11 @@ class ContextAny {
    * Value:  the store to search.
    */
   ['a']: KeyboardStore; // For 'a'ny statement.
+
+  /**
+   * If set to true, negates the 'any'.
+   */
+  ['n']: boolean|0|1;
```

Comment:
So if set to true, this becomes `notany`?

I'm also curious why include `|0|1`

Reply:
> why include `0|1`

Because those take a single character, while `true|false` take 4 and 5 characters, respectively.  It's a simple keyboard size optimization, while the `boolean` indicates the true purpose for the values.